doc/lua_api.txt

```diff
@@ -5628,6 +5628,13 @@ Environment access
       prefix `"no"` is attached, clears instead.
     * `flags` is in the same format and has the same options as `mg_flags` in
       `minetest.conf`.
+* `minetest.get_mapgen_edges([mapgen_limit[, chunksize]])`
+    * Returns the real mapgen limits in the negative and positive directions,
+      in that order.
+    * `mapgen_limit` is an optional number. If it is absent, its value is that
+      of the *active* mapgen setting `"mapgen_limit"`.
+    * `chunksize` is an optional number. If it is absent, its value is that
+      of the *active* mapgen setting `"chunksize"`.
```

Comment:
What does `*active*` mean? Is it cached until the next restart? This needs clarification for modders, at least.

Reply:
It's just terminology used elsewhere, e.g. https://github.com/minetest/minetest/blob/e84d259ec7fcc00466bd947070cb4ed77cd83c52/doc/lua_api.txt#L5631-L5632

Clarifying it is probably for another PR.